staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go

```diff
@@ -365,6 +365,13 @@ func ParseQuantity(str string) (Quantity, error) {
 		amount.Neg(amount)
 	}
 
+	copyAmount := new(inf.Dec)
```

Comment:
making a second object and doing a comparison seems like a roundabout (no pun intended) and possible expensive way of determining if we lost precision ... do we really not know if we lost precision in the parsing otherwise?

Reply:
> do we really not know if we lost precision in the parsing otherwise?

I don't know if there is a better / more performant way. Looking at the signature of the `.Round()` methods, I don't see anything that says if the value was actually rounded or not.